skbio/alignment/_tabular_msa.py

```diff
@@ -123,7 +124,12 @@ def shape(self):
         3
 
         """
-        return self._shape
+        len_sequence = len(self)
```

Comment:
Minor comment: I was confused by the names `len_sequence` as it sounds like it's the length of the sequence (it's actually the number of sequences in the MSA). Possible to rename these local vars to something like `num_sequences` and `num_positions`? or `sequence_count` and `position_count`